core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -454,6 +454,9 @@ public String getAbsoluteRemoteFs() {
         }
         @Override public Integer call() {
             Channel c = Channel.current();
+            if (c == null) {
+                return -1;
```

Comment:
I think 0 would be more appropriate, as I had in #2934. This is a count, by definition nonnegative. At any rate, the previous behavior was an NPE so this cannot be worse.